src/lib/project-saver-hoc.jsx

```diff
@@ -398,7 +398,7 @@ const ProjectSaverHOC = function (WrappedComponent) {
         vm: PropTypes.instanceOf(VM).isRequired
     };
     ProjectSaverComponent.defaultProps = {
-        autoSaveIntervalSecs: 120,
+        autoSaveIntervalSecs: 600, // 10 minutes = 600 seconds
```

Comment:
The comment is mainly here to make it easier to find this line by text search

Reply:
How dare you make things easy to find? 🤣